src/NAR/RuntimeSystem.java

```diff
@@ -0,0 +1,73 @@
+package NAR;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class RuntimeSystem {
+	
+	private static RuntimeSystem Instance = new RuntimeSystem();
+	RuntimeSystem() {}
+	
+	
+	private List listOfLines(String str) {
```

Comment:
you should specify what type of object the list returned by this method is. In your case you would return a List<String>